src/Settings/components/comon/listItemComponent.jsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { ListItem, Avatar, Icon } from "@rneui/themed";
+import PropTypes from "prop-types";
+import { useSelector } from "react-redux";
+import { styles, nightModeStyles, iconNightColor } from "../../styles";
+
+function ListItemComponent({ icon, title, value, isAvatar, actionConstant, onPressAction }) {
```

Comment:
Another function keyword which we can change to arrow body function.

Reply:
fixed